src/main/webapp/isolate/registration-form.html

```diff
@@ -0,0 +1,32 @@
+<form id="isolate-registration-form" method="POST" action="/isolate-registration">
+  <input
+      type="text"
+      name="firstname"
+      placeholder="First Name"
+      required
+      minlength="2"
+      maxlength="50"
+  >
+  <br>
+  <input
+      type="text"
+      name="lastname"
+      placeholder="Last Name"
+      required
+      minlength="2"
+      maxlength="50"
+  >
+  <br>
+  <input
+      type="text"
+      name="address"
+      placeholder="Home Address"
```

Comment:
Maybe this should be isolation address? They might not be isolating at home

Reply:
yeah that sounds good, done https://github.com/googleinterns/step226-2020/pull/4/commits/e807cc16a046fc48a2bb5529d6df3bb78e4ef7cf